src/PluginValidate/Requirements/DataformatRequirements.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * This file is part of the Moodle Plugin CI package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2024 Marina Glancy
+ * @license https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace MoodlePluginCI\PluginValidate\Requirements;
+
+use MoodlePluginCI\PluginValidate\Finder\FileTokens;
+
+/**
+ * Dataformat plugin requirements.
+ */
+class DataformatRequirements extends GenericRequirements
```

Comment:
Should we be more strict and check `classes/writer.php` existence too?

Reply:
I think if we add proper validation to the plugins, we should start with the most used ones. This PR is only about removing the false positive error